internal/app/knitgateway/version.go

```diff
@@ -21,22 +21,20 @@ import (
 	"connectrpc.com/connect"
 )
 
-const buildVersion = "v0.1.0-dev"
-
 //nolint:gochecknoglobals
 var (
-	// NB: This is a var instead of a const so it can be changed via -X ldflags.
+	// NB: These are vars instead of consts so they can be changed via -X ldflags.
+	buildVersion       = "v0.1.0"
 	buildVersionSuffix = ""
-)
 
-func Version() string {
-	return buildVersion + buildVersionSuffix
-}
+	// Version is the gateway version to report.
+	Version = buildVersion + buildVersionSuffix
```

Comment:
It turns out that this didn't need to be a function. Since the inputs are not `const`, the concatenation here is deferred to init, so the `-X` ldflag is able to change its result.